std_msgs/Header header

float64 system_time
float64 gps_time
float64 latitude
float64 longitude
float64 ellipsoidal_height
float64 velocity_north
float64 velocity_east
float64 velocity_down
float64 roll
float64 pitch
float64 heading
uint8 position_mode
uint8 velocity_mode
uint8 attitude_status

